diagnostic.md

```diff
@@ -92,7 +123,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+The font-family is cursive (inline styling... bad!) sans-serif, and the color will be yellow.
```

Comment:
You're correct in that ```font-family``` is ```cursive``` because of the inline style, but ```cursive``` and ```sans-serif```are mutually exclusive ```font-family``` values.